pkg/sql/compile/operator.go

```diff
@@ -81,22 +81,46 @@ func dupInstruction(in vm.Instruction) vm.Instruction {
 			Limit: arg.Limit,
 		}
 	case *join.Argument:
+		conds := make([][]join.Condition, len(arg.Conditions))
```

Comment:
Isn't easier to have a copyCondition function that copies arg.Conditions?

Reply:
yes